policy.go

```diff
@@ -67,6 +67,16 @@ type policyOrDataNode struct {
 
 type policyOrDataTree []policyOrDataNode
 
+type dynamicPolicyDataRaw_v0 dynamicPolicyData
```

Comment:
the order of the definitions here is a bit odd,  I think maybe a comment could help about the relation of _v0 vs dynamicPolicyData atm

same for static